packages/utils/src/lib/report.unit.test.ts

```diff
@@ -223,3 +229,37 @@ describe('getPluginNameFromSlug', () => {
     expect(getPluginNameFromSlug('plugin-b', plugins)).toBe('Plugin B');
   });
 });
+
+describe('sortAuditIssues', () => {
+  it('should sort issues by severity and source file', () => {
+    const mockIssues = [
+      { severity: 'warning', source: { file: 'b' } },
+      { severity: 'error', source: { file: 'c' } },
+      { severity: 'error', source: { file: 'a' } },
+      { severity: 'info', source: { file: 'b' } },
+    ] as Issue[];
+    const sortedIssues = [...mockIssues].sort(sortAuditIssues);
+    expect(sortedIssues).toEqual([
+      { severity: 'error', source: { file: 'a' } },
+      { severity: 'error', source: { file: 'c' } },
+      { severity: 'warning', source: { file: 'b' } },
+      { severity: 'info', source: { file: 'b' } },
+    ]);
+  });
+
+  it('should sort issues by source file and source start line', () => {
```

Comment:
We could split this test into 2 as there could be issues only maintain a file but no start line